diff --git a/CMakeLists.txt b/CMakeLists.txt
index fbab265..c773d6e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.14)
 project(limereport)
 
-set(LIMEREPORT_VERSION_MAJOR 1)
-set(LIMEREPORT_VERSION_MINOR 6)
-set(LIMEREPORT_VERSION_RELEASE 8)
+set(LIMEREPORT_VERSION_MAJOR 1)
+set(LIMEREPORT_VERSION_MINOR 7)
+set(LIMEREPORT_VERSION_RELEASE 5)
 
 option(ENABLE_ZINT "Enable libzint build for barcode support" OFF)
 option(LIMEREPORT_STATIC "Build LimeReport as static library" OFF)
